doc/api/net.md

```diff
@@ -501,7 +507,30 @@ added: v0.5.3
 
 The amount of bytes sent.
 
-### socket.connect(options[, connectListener])
+### socket.connect()
+
+Initiate the connection on a given socket.
+
+Normally this method is not needed, the socket should be created and
+opened with [`net.createConnection()`][]. Use this only if you are
+implementing a custom Socket.
+
+Possible signatures:
+
+* [socket.connect(options[, connectListener])][`socket.connect(options)`]
+* [socket.connect(path[, connectListener])][`socket.connect(path)`]
+  for connections to UNIX domain sockets or Windows named pipes.
```

Comment:
I would not *object* to that, though I think we can just say it can be used for "IPC" without mentioning the "local" part. Just like IPC is not necessarily local, but people usually assume it's talking about a local thing, readers can figure out if this API is what they are looking for by themselves.